Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG REGISTRY_PREFIX=''
+ARG CODENAME=bionic
+
+FROM ${REGISTRY_PREFIX}ubuntu:${CODENAME} as builder
+
+RUN set -x \
+	&& apt update \
+	&& apt upgrade -y \
+	&& apt install --yes --no-install-recommends \
+		build-essential \
```

Comment:
Since build only dependencies are not removed after build, the resulting container is not "minimal". The primary goal was to add matrix build support, so this might be okay?